src/features/loading.js

```diff
@@ -53,18 +56,19 @@ export const LoadingMixin = (ModelViewerElement) => {
     }
 
     get loaded() {
-      return super.loaded || this[$preloaded];
+      return super.loaded ||
+          (this.src && CachingGLTFLoader.hasFinishedLoading(this.src));
```

Comment:
This method, while probably valuable for tests at least, seems that it'd be more appropriate to live on `this.src`, our `Model` class (which could just wrap this); `CachingGLTFLoader` is an implementation detail that would benefit from continuing being hidden inside of Model, or at least out of features/*

Reply:
I don't know that I agree. `CachingGLTFLoader` was in fact the abstraction that enabled preloading to work in parallel with normal loading via setting `src`. Please offer a more concrete proposal for how you would change this!

Reply:
Keeping this as-is, but remain open to suggestions. I don't think it's the right answer to conflate preloading and the implementation in the `Model` class, because untangling that is what begot `CachingGLTFLoader` in the first place.